django/contrib/admin/static/admin/css/autocomplete.css

```diff
@@ -0,0 +1,3 @@
+select.admin-autocomplete {
```

Comment:
Not sure there's much advantage to a separate CSS file for 1 rule. Any thoughts on that?